docs/how-to/add-execution-provider.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
I suggest we remove this and ask Ryan help us write a new one based on the pluggable EP interface.

Reply:
Is the current content obsolete/not recommended? If so, I can remove the page, but if it's still useful but can just use a new update, we can leave for now and replace when we have the new content. CC @RyanUnderhill @jywu-msft